client/src/components/Workflow/Editor/Forms/FormDefault.vue

```diff
@@ -44,6 +44,15 @@
                 @input="onAnnotation"
             />
             <FormDisplay :id="id" :inputs="inputs" @onChange="onChange" />
+            <div v-if="isSubworkflow">
```

Comment:
All outputs on any node should be allowed - restricting it to tools and subworkflows is unfortunate.

Reply:
Thanks for looking into it! Yeah we could/should do that if/once the backend supports it.